### DISCUSSION TITLE
macOS 11.6.1 / latest calibre+plugin => plugin installation ok, but empty directory

### DISCUSSION THREAD

**Discussion by tschloss** [apprenticeharper/DeDRM_tools#1922] [Q&A] [answered]
Hi, in consequence the plugin does nothing. When adding an azw3 file, after a while the undecrypted book is in the the library. I am not asked for entering Kindle serials.
I installed the plugin "from file" and selected the DeDRM_plugin.zip
The plugin is listed under installed plugins.

When inspecting ~/Library/preferences/calibre/plugins  the DeDRM directory seems to be incomplete: 
- no files in "7.2.1" and "help".
- libalfcrypto.dylib in "libraryfiles"
- dedrm.json with one empty object
But a zip file DeDRM.zip is sitting there - is this correct?

I removed calibre, deleted its library older  and all again. 

Any ideas?

(I used to use the standalone app before which stopped running unfortunately. Calibre is an ugly pain!)
Thank you
Thomas

**Comment by ElleKayEm** [comment, marked as answer]
If the book is from or for an E-ink Kindle (not a Fire), then you do need to enter its serial number.  Preferences > Plugins > select DeDRM (under File type) > click the Customize plugin button > eInk Kindle ebooks > green plus sign.

**Reply by tschloss** [reply]
Thank you very much. I expected to enter the serial somewhere, but I thought a dialogue would appear when the plugin runs.
Great - it works!!

(calibre is very shy with feedback ....)

Best
Thomas
